### PR TITLE
Reexport api functions in server adapters

### LINKED ISSUE #500: Reexport `createBullBoard`, `BullAdapter` and `BullMQAdapter` in server adapters
It would be nice if `@bull-board/express` and the other server adapters would export `createBullBoard`, `BullAdapter` and `BullMQAdapter` from the `@bull-board/api` package.

This would come with the benefit to manage a little less dependencies.

The hello world example then could look like this:

```diff
const express = require('express');
const Queue = require('bull');
const QueueMQ = require('bullmq');
- const { createBullBoard } = require('@bull-board/api');
- const { BullAdapter } = require('@bull-board/api/bullAdapter');
- const { BullMQAdapter } = require('@bull-board/api/bullMQAdapter');
- const { ExpressAdapter } = require('@bull-board/express');
+ const { ExpressAdapter, createBullBoard, BullAdapter, BullMQAdapter } = require('@bull-board/express');

const someQueue = new Queue('someQueueName', {
  redis: { port: 6379, host: '127.0.0.1', password: 'foobared' },
}); // if you have a special connection to redis.
const someOtherQueue = new Queue('someOtherQueueName');
const queueMQ = new QueueMQ('queueMQName');

const serverAdapter = new ExpressAdapter();
serverAdapter.setBasePath('/admin/queues');

const { addQueue, removeQueue, setQueues, replaceQueues } = createBullBoard({
  queues: [new BullAdapter(someQueue), new BullAdapter(someOtherQueue), new BullMQAdapter(queueMQ)],
  serverAdapter: serverAdapter,
});

const app = express();

app.use('/admin/queues', serverAdapter.getRouter());

// other configurations of your server
  
app.listen(3000, () => {
  console.log('Running on 3000...');
  console.log('For the UI, open http://localhost:3000/admin/queues');
  console.log('Make sure Redis is running on port 6379 by default');
});
```

### PR DESCRIPTION
**Pull request by mmachatschek**
Fixes #500

### CODE REVIEW

File: packages/koa/src/index.ts [hunk 1]
```diff
@@ -1 +1,4 @@
 export { KoaAdapter } from './KoaAdapter';
+export { BullAdapter } from '@bull-board/api/bullAdapter';
+export { BullMQAdapter } from '@bull-board/api/bullMQAdapter';
+export { ExpressAdapter } from './ExpressAdapter';
```
Comment (felixmosh) [inline packages/koa/src/index.ts:4]: this is wrong adapter :]

File: packages/hapi/src/index.ts [hunk 2]
```diff
@@ -1 +1,4 @@
 export { HapiAdapter } from './HapiAdapter';
+export { BullAdapter } from '@bull-board/api/bullAdapter';
+export { BullMQAdapter } from '@bull-board/api/bullMQAdapter';
+export { ExpressAdapter } from './ExpressAdapter';
```
Comment (felixmosh) [inline packages/hapi/src/index.ts:4]: this is wrong adapter :]

File: packages/fastify/src/index.ts [hunk 3]
```diff
@@ -1 +1,4 @@
 export { FastifyAdapter } from './FastifyAdapter';
+export { BullAdapter } from '@bull-board/api/bullAdapter';
+export { BullMQAdapter } from '@bull-board/api/bullMQAdapter';
+export { ExpressAdapter } from './ExpressAdapter';
```
Comment (felixmosh) [inline packages/fastify/src/index.ts:4]: this is wrong adapter :]

File: README.md [hunk 4]
```diff
@@ -160,9 +157,9 @@ If you host your express service on a different path than root (/) ie. https://<
 
 ```js
 const Queue = require('bull')
-const { createBullBoard } = require('@bull-board/api')
-const { BullAdapter } = require('@bull-board/api/bullAdapter')
-const { ExpressAdapter } = require('@bull-board/express')
+const {  } = require('@bull-board/api')
```
Comment (felixmosh) [inline README.md:160]: should be removed :]

File: packages/koa/src/index.ts (hunk 1 above)
Comment (mmachatschek) [reply packages/koa/src/index.ts]: sorry about that 😅 I simultaneously answered your comment on the issue and sent in the PR fixed now

File: packages/hapi/src/index.ts (hunk 2 above)
Comment (mmachatschek) [reply packages/hapi/src/index.ts]: sorry about that 😅 I simultaneously answered your comment on the issue and sent in the PR fixed now

File: packages/fastify/src/index.ts (hunk 3 above)
Comment (mmachatschek) [reply packages/fastify/src/index.ts]: sorry about that 😅 I simultaneously answered your comment on the issue and sent in the PR fixed now

File: README.md (hunk 4 above)
Comment (mmachatschek) [reply README.md]: sorry about that 😅 I simultaneously answered your comment on the issue and sent in the PR fixed now

**Pull request merged**